src/primer.tex

```diff
@@ -124,7 +124,7 @@ \section{Bitcoin and Other Decentralized Protocols}
 The core Internet protocols are decentralized.
 Many other Internet-based protocols are similarly decentralized,
```

Comment:
In my mind both Internet and internet are valid dependent upon context. If you're referring to either "The Internet" in the sense of the high-level public network accessible on the public IP ranges, then I'd use a capital 'I'. If you're referring to internetworking in general, which seems to be the case here (I read it as internet[working] protocols) then I'd use a lower-case 'i'. If referring to Internet Protocol (IP) the standard/specification then I'd capitalize.